bazel/envoy_build_system.bzl

```diff
@@ -369,3 +374,10 @@ def envoy_select_google_grpc(xs, repository = ""):
         repository + "//bazel:disable_google_grpc": [],
         "//conditions:default": xs,
     })
+
+# Select stuff
+def envoy_select_exported_symbols(xs, repository=""):
+    return select({
+        repository + "//bazel:enable_exported_symbols": xs,
```

Comment:
I believe you can use a hardcoded `@envoy` for the repository -- the only place where that wouldn't work (the Google internal fork) rewrites this file entirely (per htuch).

Reply:
I'll give that a shot thanks